### PR TITLE
chore(element-templates): add conditional template properties docs

### PR DESCRIPTION
**Pull request by smbea**
To be merged after properties panel is released and included in the Modeler

### CODE REVIEW

File: docs/components/modeler/desktop-modeler/element-templates/defining-templates.md [hunk 1]
```diff
@@ -661,3 +666,75 @@ Per default, the element template defines the visible entries of the properties
 ```
 
 ![Display default entries](./img/entries-visible.png)
+
+### Defining Conditional Properties
+
+Properties may have a condition which determines when they should be active, depending on the value of another property.
```
Comment (MaxTru) [inline docs/components/modeler/desktop-modeler/element-templates/defining-templates.md:672]: Quick feedback.

IMO we should explain what `active` means. E.g.;

If a property is `active` it is displayed in the properties panel.
If a property is not `active` it is not displayed in the properties panel. Also the value of that property is de-serialzied from the XML.

**Review by MaxTru — CHANGES_REQUESTED** [review]

File: docs/components/modeler/desktop-modeler/element-templates/defining-templates.md (hunk 1 above)
Comment (smbea) [reply docs/components/modeler/desktop-modeler/element-templates/defining-templates.md:672]: Makes sense, I will add a sentence for this

**Comment by MaxTru** [comment]
If done, please request re-review from me. I am happy to do that then.

Please note that CI is failing

**Review by MaxTru — CHANGES_REQUESTED** [review]
This was forgotten (line 66 of the same file):

```
The application uses the `$schema` property to ensure compatibility for a given element template. The latest supported [Camunda element templates JSON Schema versions](https://github.com/camunda/element-templates-json-schema) are
- `v0.9.1` (Camunda Platform 7)
- `v0.4.1` (Camunda Platform 8)
```

**Comment by barmac** [comment]
> This was forgotten (line 66 of the same file):
> 
> ```
> The application uses the `$schema` property to ensure compatibility for a given element template. The latest supported [Camunda element templates JSON Schema versions](https://github.com/camunda/element-templates-json-schema) are
> - `v0.9.1` (Camunda Platform 7)
> - `v0.4.1` (Camunda Platform 8)
> ```

Fixed.

**Review by MaxTru — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]
Minor adjustments in latest commit to change headers into sentence case format. LGTM!

**Comment by christian-konrad** [comment]
@smbea Once we retrieve the schema in the Web Modeler, we need to either
- Move the docs out of the Desktop Modeler sections into a generic one
- Or duplicate the docs into the Web Modeler sections

Which can be challenging due to the "Element Template"/"Connector Template" name clash.
I think it makes sense to have a common section like this (note it is just a mockup)

Bildschirmfoto 2022-07-09 um 15 00 10

and have 2 additional sections for both Web and Desktop that refer to it, but with additional, product-specific details (i.e. how to include them in Desktop vs. how to write them in Web)

Can you create an issue for that?

And we should also remove this notice, because it is now part of the product we sell and thus public+reliable:

Bildschirmfoto 2022-07-09 um 14 54 13

**Review by MaxTru — APPROVED** [review]

**Comment by MaxTru** [comment]
+1 to what Christian said. Given the Connector Template feature in web modeler, we should restructure docs. Lets have a seperate issue for that.

**Comment by christian-konrad** [comment]
@smbea and @MaxTru I added an issue for the common docs : https://github.com/camunda/camunda-platform-docs/issues/1050

**Pull request merged**